ui/app/templates/components/namespace-picker.hbs

```diff
@@ -1,119 +1,26 @@
-                (object-at (dec 2 this.menuLeaves.length) this.lastMenuLeaves)
-                this.auth.authData.userRootNamespace
-              }}
-              @class="namespace-link is-current button is-transparent icon"
-            >
-              <div class="is-flex-align-baseline">
-                <Hds::Button
-                  @text="Go back"
-                  @icon="chevron-left"
-                  @isIconOnly={{true}}
-                  @color="tertiary"
-                  class="is-flex-align-baseline"
-                />
-                <p class="is-size-8 has-text-grey has-text-weight-semibold is-uppercase">Namespaces</p>
-              </div>
-            </NamespaceLink>
-          {{/if}}
-        </div>
-        {{#if (includes "" this.lastMenuLeaves)}}
-          {{! leaf is '' which is the root namespace, and then we need to iterate the root leaves }}
-          <div class="leaf-panel {{if (eq '' this.currentLeaf) 'leaf-panel-current' 'leaf-panel-left'}} ">
-            {{#each this.rootLeaves as |rootLeaf|}}
-              <NamespaceLink @targetNamespace={{rootLeaf}} @class="namespace-link" @showLastSegment={{true}} />
-            {{/each}}
-          </div>
-        {{/if}}
-        {{#each this.lastMenuLeaves as |leaf|}}
-          {{#if leaf}}
-            <div
-              class="leaf-panel
-                {{if (eq leaf this.currentLeaf) 'leaf-panel-current' 'leaf-panel-left'}}
-                {{if (and this.isAdding (eq leaf this.changedLeaf)) 'leaf-panel-adding'}}
-                {{if (and (not this.isAdding) (eq leaf this.changedLeaf)) 'leaf-panel-exiting'}}
-                "
-            >
-              {{#each-in (get this.namespaceTree leaf) as |leafName|}}
-                <NamespaceLink
-                  @targetNamespace={{concat leaf "/" leafName}}
-                  @class="namespace-link"
-                  @showLastSegment={{true}}
-                />
-              {{/each-in}}
-            </div>
-          {{/if}}
-        {{/each}}
-        {{#if this.canList}}
-          <div class="leaf-panel leaf-panel-current">
-            <div class="level">
-              <span class="level-left">
-                <LinkTo @route="vault.cluster.access.namespaces" class="is-block namespace-link namespace-manage-link">
-                  Manage Namespaces
-                </LinkTo>
-              </span>
-              <span class="level-right">
-                <Hds::Button
-                  @text="Refresh namespace list"
-                  @icon="reload"
-                  @isIconOnly={{true}}
-                  @color="tertiary"
-                  data-test-refresh-namespaces
-                  {{on "click" (action "refreshNamespaceList")}}
-                />
-              </span>
-            </div>
-          </div>
-        {{/if}}
-      </div>
-    </D.Content>
-  </BasicDropdown>
+  <Hds::Form::SuperSelect::Single::Field
+    @ariaLabel="Namespace"
+    @label="Namespace"
```

Comment:
How does the HTML for this render? Since this input doesn't have a `@label` I _think_  just including `@ariaLabel` should be sufficient 🤔

Reply:
just checked and you're right - it doesn't render anything, removed ✔️